region/norfair/east/Double Chamber.json

```diff
@@ -639,12 +639,9 @@
     },
     {
       "link": [3, 4],
-      "name": "Double Chamber Spike IBJ",
+      "name": "Spike IBJ",
       "requires": [
-        "h_canNavigateHeatRooms",
-        "canIframeSpikeJump",
-        {"spikeHits": 1},
-        "h_canIBJ",
+        "h_IBJFromSpikes",
```

Comment:
heated version of the helper?  Since it has that `or` which I think means number of attempts.